src/Resources/views/main.html.twig

```diff
@@ -1,15 +1,15 @@
 {% extends "@ContaoInstallation/layout.html.twig" %}
 
 {% block main %}
-  <fieldset class="tl_tbox{% if sql_form is empty and sql_message is empty %} collapsed{% endif %}">
+  <fieldset class="tl_tbox{% if sql_form is empty and sql_message is defined and sql_message is empty %} collapsed{% endif %}">
     <legend onclick="this.getParent().toggleClass('collapsed')">{{ 'update_tables'|trans }}</legend>
     <div>
       <h3>{{ 'update_tables'|trans }}</h3>
       {% if sql_form is empty %}
         <p class="tl_confirm">{{ 'update_tables_confirm'|trans }}</p>
       {% endif %}
       <p>{{ 'update_tables_explain'|trans|raw }}</p>
-      {% if sql_message is not empty %}
+      {% if sql_message is defined and sql_message is not empty %}
```

Comment:
As mentioned in the original bug report, you can shorten
```twig
{% if sql_message is defined and sql_message is not empty %}
```
by using
```twig
{% if sql_message|default %}
```
It's recommended to use as little logic as possible in Twig templates.

Reply:
Good point, thank you! (Didn't see the bug is already reported (#72)).